src/robotto.js

```diff
@@ -82,4 +82,44 @@ robotto.parse = function(robotsFile) {
     return rulesObj;
 };
 
+robotto._stripRoute = function(urlParam) {
+    // Removes domain name and returns only the endPoint
+    return urlParam.replace(/^.*\/\/[^\/]+/, '');
+};
+
+robotto.check = function(userAgent, urlParam, rulesObj) {
+    let userAgents = Object.keys(rulesObj);
+    let desiredRoute = this._stripRoute(urlParam) + '/';
+    let allowed = true;
+
+    // Searches for every user agent until it gets a match
+    userAgents.forEach((agent) => {
+        if (agent.indexOf(userAgent) === 0) {
```

Comment:
It should be true for a partial match, shouldn't it?
I think this is the expected behavior, if a website has `/browse/` as disallowed, trying to crawl `/browse/whatever` should be disallowed too, shouldn't it?
That's why some people use `/` as disallowed to disable crawling of the whole website.

I agree on `toLowerCase` though.